docs/resources_migrated.md

```diff
@@ -32,6 +32,7 @@ service catalog resources or OLM resources, OLM migration is not handled by Cran
   - serviceplans
   - operatorgroups
   - events
+  - events.events.k8s.io
```

Comment:
Does this need to be documented in downstream docs?

Reply:
@apinnick Ahh, good question. If downstream docs include the same content here -- i.e. listing the default set of excluded resources -- then probably yes. The below is the full list (with the last in the list being the new one just added):
```
  - imagetags
  - templateinstances
  - clusterserviceversions
  - packagemanifests
  - subscriptions
  - servicebrokers
  - servicebindings
  - serviceclasses
  - serviceinstances
  - serviceplans
  - operatorgroups
  - events
  - events.events.k8s.io
```

Reply:
Yes, it looks like [downstream docs](https://docs.openshift.com/container-platform/4.8/migration_toolkit_for_containers/advanced-migration-options-mtc.html#migration-excluding-resources_advanced-migration-options-mtc) includes the full list: 

  excluded_resources: 
  - imagetags
  - templateinstances
  - clusterserviceversions
  - packagemanifests
  - subscriptions
  - servicebrokers
  - servicebindings
  - serviceclasses
  - serviceinstances
  - serviceplans
  - operatorgroups

I'll create a bug to add the new resource.
  - events

Reply:
@sseago 
Hi, I am updating the relevant downstream doc. Does this change apply to 4.6+?